packages/adapters/solana/src/client.ts

```diff
@@ -187,7 +188,7 @@ export class SolanaAdapter extends AdapterBlueprint {
     )
 
     return {
-      signature: new TextDecoder().decode(signature)
+      signature: bs58.encode(signature)
```

Comment:
are we changing signature schemes?

Reply:
no this decoding was originally wrong, but we were not using it before. This was decoding bytes into utf-8 which would result in wrong characters, later was not possible to verify the signature.